solr/core/src/java/org/apache/solr/cloud/OverseerConfigSetMessageHandler.java

```diff
@@ -98,6 +99,8 @@ public OverseerSolrResponse processMessage(ZkNodeProps message, String operation
         log.error("ConfigSet: {} operation: {} failed", configSetName, operation, e);
       }
 
+      addExceptionToNamedList(operation, e, results);
+
       results.add("Operation " + operation + " caused exception:", e);
```

Comment:
Do we still need this after adding the `addExceptionToNamedList` call?

Reply:
Nope, I had forgotten to remove the original code in this instance. I did remove it